app/src/main/res/values-pl/strings.xml

```diff
@@ -54,24 +120,57 @@
     <string name="synchronize">Synchronizuj</string>
     <string name="set_bookmark_name">Wpisz nazwę zakładki</string>
     <string name="set_bookmark_at">Ustaw zakładkę w</string>
-    <string name="no_bookmarks">Nie ma zakładek dla tego pliku audio</string>
+    <string name="no_bookmarks">Nie ma jeszcze zakładek dla tego pliku audio</string>
     <string name="bookmarks">Zakładki</string>
+    <string name="untitled_bookmark">Niezatytułowana</string>
+    <string name="delete_from_device">Usuń z urządzenia</string>
     <string name="remove_from_db">Usuń z bazy danych</string>
+    <string name="remove_from_library">Usuń z biblioteki</string>
     <string name="mark_as_not_started">Oznacz jako nie rozpoczęty</string>
     <string name="mark_as_completed">Oznacz jako ukończony</string>
-    <string name="items_selected">%d zaznaczonych</string>
+    <plurals name="items_selected">
+        <item quantity="one">%d zaznaczona/y</item>
+        <item quantity="other">%d zaznaczone/ych</item>
```

Comment:
Here I get an additional error message for the missing quantity *few* (and for the remaining plural strings). Again, I don't know if it makes a difference here in Polish. So if you had a good reason for leaving it out I'll merge it anyway.